flake8_quotes/__init__.py

```diff
@@ -10,33 +10,57 @@ class QuoteChecker(object):
     version = __version__
 
     QUOTES = {
-        # When user wants only single quotes
+        # Allow single quotes
+        # Fail on double quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'allow': '\'',
+            'fail': '"',
         },
+
+        # Allow double quotes
+        # Fail on single quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'allow': '"',
+            'fail': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
+        # Allow single multiline quotes
+        # Fail on double multiline quotes
+        '\'': {
+            'allow': '\'\'\'',
+            'fail': '"""'
+        },
+
+        # Allow double multiline quotes
+        # Fail on single multiline quotes
+        '"': {
+            'allow': '"""',
+            'fail': '\'\'\''
+        }
+    }
+
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append('quotes')
+        parser.add_option('--multiline-quotes', default='\'', action='store',
+                          help='Multiline quote to expect in all files (default: \')')
+        parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        if hasattr(options, 'multiline_quotes'):
+            multiline_quotes = options.multiline_quotes
+        else:
+            multiline_quotes = options.quotes
+
         cls.quotes = cls.QUOTES[options.quotes]
+        cls.multiline_quotes = cls.MULTILINE_QUOTES[multiline_quotes]
```

Comment:
Is this related to the bug I noted in the PR description? If so I was considering fixing it as modifying the tests felt a bit dirty and I thought it might break some stuff for existing users. I'll add that option now.